src/main/java/io/vertx/ext/dropwizard/impl/Matcher.java

```diff
@@ -15,14 +14,26 @@
 
```

Comment:
Hey @tsegismont first of all thank you for your review and comments.
Yep you are right there is an easy way how to reduce the amount of fields inside Matcher class. Also we can easily use nulls instead of Optionals. 
But what about the need to perform matching second time in `Matcher.matchIdentifier` method. I see a couple of ways how to avoid this but all of them looks a bit ugly to me. One of them would be to change the signature of `Matcher.match` method, so it might return String instead of boolean. In case if there is no match it would return null in other case the value field from `Match` instance that we can map to the identifier later without second loop of matching.
What do you think about it?